dockers/docker-lldp-sv2/lldpd.conf.j2

```diff
@@ -0,0 +1,3 @@
+{% for member in alias_map %}
+configure ports {{member['sonic']}} lldp portidsubtype local {{member['hardware']}}
```

Comment:
it is not hardware name, it is ngs interface name. maybe we can name it as "origin"?